requirements.txt

```diff
@@ -0,0 +1,84 @@
+-i https://pypi.org/simple
+alabaster==0.7.13; python_version >= '3.6'
+babel==2.13.0; python_version >= '3.7'
+blinker==1.6.3; python_version >= '3.7'
+certifi==2023.7.22; python_version >= '3.6'
+charset-normalizer==3.3.0; python_full_version >= '3.7.0'
+chemprop==1.6.1; python_version >= '3.7'
+click==8.1.7; python_version >= '3.7'
+cloudpickle==3.0.0; python_version >= '3.8'
+contourpy==1.1.1; python_version >= '3.8'
+cycler==0.12.1; python_version >= '3.8'
+descriptastorus==2.6.1; python_version == '3.8'
+docstring-parser==0.15; python_version >= '3.6' and python_version < '4.0'
+docutils==0.20.1; python_version >= '3.7'
+filelock==3.12.4; python_version >= '3.8'
+flask==3.0.0; python_version >= '3.8'
```

Comment:
All this libraries are really needed?

Reply:
These are all dependencies of the core libraries that are listed in requirements. They're all listed here so `pip` can correctly resolve dependencies.